src/doc/unstable-book/src/language-features/debugger-visualizer.md

```diff
@@ -14,12 +14,8 @@ to embed a debugger visualizer file into the PDB/ELF generated by `rustc`.
 ``` rust,ignore (partial-example)
 #![feature(debugger_visualizer)]
 #![debugger_visualizer(natvis_file = "foo.natvis")]
+#![debugger_visualizer(gdb_script_file = "foo.py")]
 struct Foo {
 
 }
 ```
-
-## Limitations
-
-Currently, this feature only supports embedding Natvis files on `-windows-msvc`
```

Comment:
The `msvc` part is still true right? It doesn't seem like this would work with the `-windows-gnu` targets so worth keeping language along those lines.